tests/test_classification_metric.py

```diff
@@ -108,3 +109,82 @@ def test_between_group():
 
     b = np.array([0.5, 0.5, 1.25, 1.25, 1.25, 1.25])
     assert cm.between_group_generalized_entropy_index() == 1/12*np.sum(b**2 - 1)
+    
+def test_multiclass_confusion_matrix():
+    data = np.array([[0, 1],
+                     [0, 0],
+                     [1, 0],
+                     [1, 1],
+                     [1, 0],
+                     [1, 2],
+                     [2, 1],
+                     [2, 0],
+                     [2, 2],
+                     [2, 1]])
+    pred = data.copy()
+    pred[3,1] = 0
+    pred[4,1] = 2
+    
+    df = pd.DataFrame(data, columns=['feat', 'label'])
+    df2 = pd.DataFrame(pred, columns=['feat', 'label'])
+
+    favorable_values = [0,1]
+    unfavorable_values = [2]
+    bld = MulticlassLabelDataset(favorable_label = favorable_values, unfavorable_label = unfavorable_values , df = df , label_names=['label'],
+        protected_attribute_names=['feat'])
+    bld2 = MulticlassLabelDataset(favorable_label = favorable_values, unfavorable_label = unfavorable_values , df=df2, label_names=['label'],
+        protected_attribute_names=['feat'])
+    cm = ClassificationMetric(bld, bld2, unprivileged_groups=[{'feat': 2}],
+        privileged_groups=[{'feat': 0},{'feat': 1}])
+    confusion_matrix = cm.binary_confusion_matrix()
+    assert confusion_matrix == {'TP': 7.0, 'FN': 1.0, 'TN': 2.0, 'FP': 0.0}
+
```

Comment:
can you also add an assert to make sure the original labels are preserved by `ClassificationMetric`?